csharp/dapper/DapperORM/DapperORM.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.90" />

```

Comment:
Since this PR was created almost a year ago, should we revisit the versions of the packages being used and switch to the latest, if possible and preferred?